src/pybind/mgr/tox.ini

```diff
@@ -157,7 +157,7 @@ commands =
 [testenv:flake8]
 basepython = python3
 deps =
-    flake8
+    flake8==6.1.0
```

Comment:
or maybe `>6.0 & <7.0`! let me know your thoughts. @adk3798

Reply:
I personally like `>6.0,<7.0` as you should in theory continue to get bugfixes but not major behavioral changes... at least not intended ones ;-)

Reply:
yesterday i talked with @adk3798 and for now we decided to go without pinning into anything. Since this was a smaller change.